src/placeos-driver/core_ext.cr

```diff
@@ -19,3 +19,45 @@ class Object
     fallback
   end
 end
+
+# Temporary fix for array shift / pop issue on crystal 1.0.0
```

Comment:
I should probably wrap this in a macro checking the crystal lang version
i.e. version 1 and below